tests/e2e/etcd_mix_versions_test.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2022 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"go.etcd.io/etcd/client/pkg/v3/fileutil"
+	"go.etcd.io/etcd/tests/v3/framework/config"
+	"go.etcd.io/etcd/tests/v3/framework/e2e"
+)
+
+func TestMixVersionsSendSnapshot(t *testing.T) {
+	cases := []struct {
+		name              string
+		clusterVersion    e2e.ClusterVersion
+		newInstaceVersion e2e.ClusterVersion
+	}{
+		// etcd doesn't support this scenario. For example, etcd cluster
```

Comment:
This is still incorrect as I mentioned in comment above. I would be ok if you marked it as TODO.

Reply:
Updated comment and also added TODO comment.

Reply:
PTAL